chainer/functions/__init__.py

```diff
@@ -170,12 +171,16 @@
 det = det.det
 Exp = exponential.Exp
 exp = exponential.exp
+Expm1 = exponential_m1.Expm1
+expm1 = exponential_m1.expm1
 Identity = identity.Identity
 identity = identity.identity
 Inv = inv.Inv
 inv = inv.inv
 Log = exponential.Log
 log = exponential.log
+Log1p = exponential_m1.Log1p
+log1p = exponential_m1.log1p
```

Comment:
The other alternative is to include `log1p` and `expm1` to `exponential`, too, though I am not certain this is better choice.